fastestimator/util/latex_util.py

```diff
@@ -36,6 +36,54 @@ def dumps(self) -> str:
         return self.dumps_content()
 
 
+class TabularCell(Container):
+    """A class to represent a tabularx cell that can directly used in Tabular.add_row(). The purpose of this class is to
```

Comment:
our doc-string convention is the first line needs to be short enough to not wrap to the next line. more details are given in second paragraph

Reply:
fixed